server/expression.js

```diff
@@ -252,6 +253,7 @@ expression.getTfaProfile_ = function(fileName, geneNames, conditionNames) {
   });
   console.log('returning TFA line', geneNames.join(','));
   return {
+    fileName: fileName,
```

Comment:
Why does the server need to return fileName? If you send a request from client then the client knows the fileName right?